### PR TITLE
16364 Changed store actions type

### PR DESCRIPTION
**Pull request by bennypc**
*Issue #:* /bcgov/entity#16364

*Description of changes:*
Changed action store type to match new requirements

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList.vue [hunk 1]
```diff
@@ -84,7 +84,7 @@ import { Action, Getter } from 'pinia-class'
 import { AddCommentDialog, DownloadErrorDialog, FileCorrectionDialog, LoadCorrectionDialog }
   from '@/components/dialogs'
 import { CorrectionTypes } from '@/enums'
-import { ActionBindingIF, ApiFilingIF, CorrectionFilingIF } from '@/interfaces'
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList.vue:87]: As this IF is no longer used (and the other one in that file is not used), please delete it.

**Review by severinbeauvais — APPROVED** [review]
Looks good. Remember to update the app version and commit the package files.

**Review by leodube-aot — APPROVED** [review]
LGTM, just the one comment from Sev about removing the unused interfaces. For my similar ticket I just had to remove ActionIF and ActionBindingIF from action-interface.ts.

**Review by JazzarKarim — APPROVED** [review]
LGTM 👍

File: src/components/Dashboard/FilingHistoryList.vue (hunk 1 above)
Comment (bennypc) [reply src/components/Dashboard/FilingHistoryList.vue:87]: Which IF are you referring to?

**Comment by ChrisGabel23** [comment]
When I did a similar ticket for Create-UI, there was a file with the action interfaces that was no longer needed. I don't see that file being deleted here: src/interfaces/action-interface.ts

File: src/components/Dashboard/FilingHistoryList.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList.vue:87]: The one you stopped using in this ticket.

File: src/interfaces/action-interface.ts [hunk 2]
```diff
@@ -4,8 +4,3 @@ import { ActionContext } from 'vuex'
 export interface ActionIF {
```
Comment (ChrisGabel23) [inline src/interfaces/action-interface.ts:4]: Is the "ActionIF" interface being used anywhere? This is the file @severinbeauvais was talking about deleting.

File: src/components/Dashboard/FilingHistoryList.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList.vue:87]: Please also remove the reference to this file from the interfaces/index.ts file. It's causing a failure of the build and unit tests checks.

Before committing, you should do these to check for issues:
```
npm run lint
npm run test:unit
```

File: src/interfaces/action-interface.ts (hunk 2 above)
Comment (severinbeauvais) [reply src/interfaces/action-interface.ts]: Please also remove the reference to this file from the interfaces/index.ts file. It's causing a failure of the build and unit tests checks.

Before committing, you should do these to check for issues:
```
npm run lint
npm run test:unit
```

Comment (bennypc) [reply src/interfaces/action-interface.ts]: removed reference!

**Pull request merged**